### ISSUE TITLE
Front End e Web Designer é a mesma coisa?

### ISSUE THREAD

**Issue by woliveiras** [woliveiras/FFAQ#2]

**Comment by VictorHugooc** [comment]
Não, apesar de ambas as áreas conversarem bastante e serem próximas, o web design define a aparência que o site deve ter, trabalha nas imagens que serão utilizadas de forma a garantir que tudo vai se encaixar da melhor maneira possível, enquanto o front end vai pegar a ideia o layout e transformar no site de fato, vai pegar o que lhe foi entregue pelo web designer e através de HTML e CSS vai gerar o site, e com Javascript vai torna-lo interativo tornando o site algo vivo e não só uma "imagem estática".
